mods/cnc/weapons/missiles.yaml

```diff
@@ -197,62 +137,40 @@ MammothMissiles:
 			Wood: 60
 			Light: 100
 			Heavy: 50
-		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Structure, Wall, Husk, Trees, Creep
 	Warhead@3Eff: CreateEffect
 		Explosions: med_frag
 		ImpactSounds: xplos.aud
 
 227mm.stnk:
+	Inherits: ^MissileWeapon
 	ReloadDelay: 70
 	Range: 7c0
 	Report: rocket1.aud
 	Burst: 2
 	BurstDelay: 10
-	ValidTargets: Ground, Air
 	Projectile: Missile
-		Arm: 0
-		Blockable: false
-		Inaccuracy: 213
-		Image: DRAGON
 		HorizontalRateOfTurn: 10
-		TrailImage: smokey
-		ContrailLength: 8
 		Speed: 213
 		RangeLimit: 8c409
 	Warhead@1Dam: SpreadDamage
-		Spread: 128
 		Damage: 60
-		ValidTargets: Ground, Air
 		Versus:
 			None: 25
 			Wood: 75
 			Light: 100
 			Heavy: 90
-		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Structure, Wall, Husk, Trees, Creep
-	Warhead@3Eff: CreateEffect
-		Explosions: small_frag
-		ImpactSounds: xplos.aud
 
 BoatMissile:
+	Inherits: ^MissileWeapon
```

Comment:
This also introduces `ValidTargets: Ground, Air`.  Not sure if this changes anything in practice.